PatientWebApplication/UserMicroserviceApi/Model/Shift.cs

```diff
@@ -0,0 +1,33 @@
+/***********************************************************************
+ * Module:  Shift.cs
+ * Author:  Luna
+ * Purpose: Definition of the Class Employee.Shift
+ ***********************************************************************/
+
+namespace UserMicroserviceApi.Model
+{
+    public class Shift : Entity
+    {
+        public string startTime { get; set; }
+        public string endTime { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).